action-run.sh

```diff
@@ -15,10 +15,13 @@ if [ "${CI}" = "true" ]; then
 fi
 
 ## Freeing space since 14GB are not enough anymore
-df -h
-echo "Deleting android, dotnet, haskell, CodeQL, Python, swift to free up space"
-sudo rm -rf /usr/local/lib/android /usr/share/dotnet /usr/local/.ghcup /opt/hostedtoolcache/CodeQL /opt/hostedtoolcache/Python /usr/share/swift
-df -h
+if [ "${CI}" = "true" ]; then
+  df -ih
```

Comment:
NIT: If you are leaving the `df -hi` at the end you might want to have it also here so you can know the amount of data used by the run.